### PR TITLE
Updating documentation markdown files (JOSS)

### PR DESCRIPTION
**Pull request by jschlenger**
Updating the documentation based on the JOSS publication and the software archive on Zenodo.

### CODE REVIEW

**Comment by pjanck** [comment]
- please change `oip.bib` appropriately [here](https://github.com/tumcms/Open-Infra-Platform/tree/development/Documentation)
- add the PDF of the paper next to the existing one of Hecht & Jaud

File: Documentation/oip.bib [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
\ No newline at end of file
+﻿@article{Jaud2022,
+  doi = {10.21105/joss.03061},
+  url = {https://doi.org/10.21105/joss.03061},
+  year = {2022},
+  publisher = {The Open Journal},
+  volume = {7},
+  number = {72},
+  pages = {3061},
+  author = {Štefan Jaud and Helge Hecht and Jonas Schlenger and Julian Amann},
```
Comment (pjanck) [inline Documentation/oip.bib]:
```suggestion
  author = {Jaud, {\v{S}}tefan and Hecht, Helge and Schlenger, Jonas and Amann, Julian},
```

See also https://github.com/openjournals/joss/issues/1069

**Review by pjanck — COMMENTED** [review]
Thanks for the update!

File: Documentation/oip.bib (hunk 1 above)
Comment (jschlenger) [reply Documentation/oip.bib]: Resolved: [f6634e9](https://github.com/tumcms/Open-Infra-Platform/pull/545/commits/f6634e9aa45ac61a90048080a7b1fd16d8a7f423)

**Review by pjanck — COMMENTED** [review]
Anything else still to come? Otherwise, I'd approve.

**Pull request merged**